communication/CMakeLists.txt

```diff
@@ -20,14 +20,11 @@ include_directories(${PROJECT_SOURCE_DIR}/communication/include)
 
 file(GLOB SOURCES "src/*.cpp")
 
-find_package(PkgConfig REQUIRED)
-pkg_check_modules(JSONCPP jsoncpp)
-
-add_executable(broadcaster ${SOURCES})
-target_link_libraries(broadcaster
+add_executable(communication ${SOURCES})
+target_link_libraries(communication
     pthread
     tcp
-    ${JSONCPP_LIBRARIES}
+    jsoncpp_lib
     librobot
 )
-install(TARGETS broadcaster DESTINATION bin)
+install(TARGETS communication DESTINATION bin)
```

Comment:
jeg har kaldt den com_module.

Reply:
Not relevant. Det er fordi master ikke er merget ind, men det blev omdøbet på `static_config`-branchen.